### ISSUE TITLE
Support for WebKit's delegtes

### ISSUE THREAD

**Issue by cojoj** [RxSwiftCommunity/RxWebKit#10]
Right now `RxWebKit` supports _only_ properties taken directly from `WKWebView`. When I started playing around more extensively with `WebKit` I started missing stuff from `WKNavigationDelegate` and `WKUIDelegate`.
Is there any specific reason why `RxWebKit` doesn't implement proxies for those delegates, or is it only because you didn't need them?
I'd love to see _official_ implementation for those proxy delegates 😉 If you're willing to add them I'd be happy to implement them! Just let me know 🙌

**Comment by freak4pc** [comment]
Hey @cojoj , If you feel something is missing feel free to push a PR ! I'll help review.

**Comment by cojoj** [comment]
Haha, it's not that simple as I thought it to be...
When I started with `WKNavigationDelegate` it turned out that there're some methods that _need_ to be called implemented, because they have completion handler and, so they wait for some default value. In that case our proxy has to provide default implementation for those.
For example here: https://github.com/ReactiveX/RxSwift/blob/master/RxCocoa/iOS/Proxies/RxPickerViewDataSourceProxy.swift
they have something called `PickerViewDataSourceNotSet` and it's a _default_ implementation.
I guess in order to make delegates for `WKWebView` work we need to have something similar since I on every run I get the following crash:
```
Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Completion handler passed to -[NSObject webView:decidePolicyForNavigationAction:decisionHandler:] was not called'
```
It's really strange that `WKNavigationDelegate` crashes, but `WKUIDelegate` works fine, even though it's methods also have completion handlers.

**Comment by bobgodwinx** [comment]
@mokumoku @freak4pc @rynecheow is someone working on this issue? Can I fix? it because I have it working locally already? the delegate closure is a real pain.

**Comment by rynecheow** [comment]
Can you submit a pull request @bobgodwinx ?

**Comment by bobgodwinx** [comment]
@rynecheow I saw you were working on it on `update/rxswift4` I mean you were on the good path though. I will take it from the branch?

**Comment by bobgodwinx** [comment]
fixed in #15
